android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterStatusProvider.java

```diff
@@ -129,17 +129,19 @@ private boolean bindToService(){
 		Intent bindingIntent = new Intent();
 		bindingIntent.setClassName(this.routerService.getPackageName(), this.routerService.getClassName());//This sets an explicit intent
 		//Quickly make sure it's just up and running
-		if(Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
-			context.startService(bindingIntent);
-		}else {
-			bindingIntent.putExtra(FOREGROUND_EXTRA, true);
-			context.startForegroundService(bindingIntent);
+		if (!bindOnly) {
```

Comment:
These changes were not discussed nor agreed upon in the proposal. Please remove changes to this file as it is not part of the proposal changes.